src/app/pages/send/send-crypto-asset-form/form/stx/stx-send-form.tsx

```diff
@@ -2,14 +2,16 @@ import { StxAvatar } from '@app/components/crypto-assets/stacks/components/stx-a
 import { useCryptoCurrencyMarketData } from '@app/query/common/market-data/market-data.hooks';
 
 import { AmountField } from '../../components/amount-field';
-import { SelectedAssetField } from '../../components/selected-asset-field';
+import { SelectedAsset } from '../../components/selected-asset';
 import { SendFiatValue } from '../../components/send-fiat-value';
 import { SendMaxButton } from '../../components/send-max-button';
 import { StacksCommonSendForm } from '../stacks/stacks-common-send-form';
 import { useStxSendForm } from './use-stx-send-form';
 
```

Comment:
Lets make an asset symbol token for these higher up
`type assetSymbol = 'BTC' | 'STX'`